shared/src/bundles/user-details/validation-schemas/user-details-create.validation-schema.ts

```diff
@@ -64,6 +64,8 @@ const userDetailsCreate = joi.object<UserDetailsCreateRequestDto, true>({
     companyWebsite: joi.string().trim().uri(),
     employerPosition: joi.string().trim(),
     cvId: joi.string().trim(),
+    talentBadges: joi.array().items(joi.string().trim()),
```

Comment:
talentBadges: joi
        .array()
        .items(
            joi.object({
                userId: joi.string().required(),
                score: joi.number().required(),
                level: joi.string().required(),
                badgeId: joi.string().required(),
            }),
        )

Reply:
as I understood @MaksymGavatuk about talent badges, we should recieve from frontend only array of strings = Ids

Reply:
Yes it just string[]